nextflow_schema.json

```
@@ -871,6 +870,16 @@
                     "fa_icon": "fas fa-cog",
                     "hidden": true
                 },
+                "multiqc_samples": {
+                    "type": "string",
+                    "exists": true,
+                    "format": "file-path",
+                    "description": "Path to a file containing internal ids and customer ids in tab seperated format. For more info check, https://docs.seqera.io/multiqc/reports/customisation#sample-name-replacement",
```

```suggestion
                    "description": "Path to a file containing internal ids and alternative ids in tab separated format. Will replace the internal id with the alternative id in the report. For more info check, https://docs.seqera.io/multiqc/reports/customisation#sample-name-replacement",
```
Just a suggestion to make it a bit  clearer